common/lib/xmodule/xmodule/tests/test_sequence.py

```diff
@@ -58,38 +55,59 @@ def _set_up_course_xml(cls):
         xml.SequenceFactory.build(  # sequence_4_1
             parent=chapter_4,
             hide_after_due=str(True),
-            due=str(cls.TOMORROW),
+            due=str(DUE_DATE),
         )
 
         for _ in range(3):
             xml.VerticalFactory.build(parent=sequence_3_1)
 
         return course
 
-    @classmethod
-    def _set_up_block(cls, parent, index_in_parent):
+    def _set_up_block(self, parent, index_in_parent):
         """
         Sets up the stub sequence module for testing.
         """
         block = parent.get_children()[index_in_parent]
 
-        cls._set_up_module_system(block)
+        self._set_up_module_system(block)
 
         block.xmodule_runtime._services['bookmarks'] = Mock()  # pylint: disable=protected-access
         block.xmodule_runtime._services['user'] = StubUserService()  # pylint: disable=protected-access
         block.xmodule_runtime.xmodule_instance = getattr(block, '_xmodule', None)  # pylint: disable=protected-access
         block.parent = parent.location
         return block
 
-    @classmethod
-    def _set_up_module_system(cls, block):
+    def _set_up_module_system(self, block):
         """
         Sets up the test module system for the given block.
         """
         module_system = get_test_system()
         module_system.descriptor_runtime = block._runtime  # pylint: disable=protected-access
         block.xmodule_runtime = module_system
 
+    def _get_rendered_student_view(self, sequence, requested_child=None, extra_context=None, self_paced=False):
+        """
+        Returns the rendered student view for the given sequence and the
+        requested_child parameter.
+        """
+        context = {'requested_child': requested_child}
+        if extra_context:
+            context.update(extra_context)
+
+        # The render operation will ask modulestore for the current course to get some data. As these tests were
+        # originally not written to be compatible with a real modulestore, we've mocked out the relevant return values.
+        with patch.object(SequenceModule, '_get_course') as mock_course:
+            type(mock_course.return_value).end = PropertyMock(return_value=COURSE_END_DATE)
```

Comment:
Nit: Instead of mocking individual properties, why not just have `mock_course.return_value = self.course`?